app/api/csv/csvExporter.ts

```diff
@@ -0,0 +1,217 @@
+import { Writable } from 'stream';
+import { EventEmitter } from 'events';
+import * as csv from '@fast-csv/format';
+import templates from 'api/templates';
+import { PropertySchema } from 'shared/types/commonTypes';
+import { TemplateSchema } from 'shared/types/templateType';
+import translate, { getLocaleTranslation, getContext } from 'shared/translate';
+import translations from 'api/i18n/translations';
+import formatters from './typeFormatters';
+
+export type SearchResults = {
```

Comment:
Everything is exported: all functions, all types.  Is this really necessary? The specs only import about 12 of the 16 exports here, and external modules should probably only import the default.

Reply:
You're right. It also suggests that the testing strategy might be wrong. I propose you not exporting the elements that are not explicitly tested right away, and assuming the tech debt of refactoring this class.